model/otel/sampling.yaml

```diff
@@ -0,0 +1,49 @@
+groups:
+  - id: sampling.common
+    type: attribute_group
+    brief: 'Semantic conventions describing sampling in OpenTelemetry.'
+    prefix: sampling
+    attributes:
+      - id: priority
```

Comment:
we try to put all attributes into the registry - please use `url.full` as an example.
Then individual conventions can refer to that attribute

Reply:
Done!